xen/include/xen/device_tree.h

```diff
@@ -764,6 +764,25 @@ int dt_parse_phandle_with_args(const struct dt_device_node *np,
                                const char *cells_name, int index,
                                struct dt_phandle_args *out_args);
 
+/**
+ * dt_count_phandle_with_args() - Find the number of phandles references in a property
```

Comment:
I don't want to touch the original comment at all. It is a port from Linux as is.